installing/installing_aws/installing-aws-account.adoc

```diff
@@ -16,6 +16,8 @@ include::modules/installation-aws-permissions.adoc[leveloffset=+1]
 
 include::modules/installation-aws-iam-user.adoc[leveloffset=+1]
 
+include::modules/manually-removing-cloud-creds.adoc[leveloffset=+1]
```

Comment:
I think the module looks fine, but it should go in the post-installation steps instead of this assembly. This one is an installation prereq.

Reply:
It might also be worth adding the xrefs to the new, post installation location to the relevant assemblies, like in the 4.7+ PR.